ReelTalk/containers/Lolomo.js

```diff
@@ -24,12 +24,9 @@ var Lolomo = React.createClass({
 
   _showDetails: function(show) {
     this.props.navigator.push({
-      title: show.name,
+      title: 'Show',
```

Comment:
I was under the impression that show was the blanket term for either movie or TV show.